2_process/src/tally_site_obs.R

```diff
@@ -1,6 +1,7 @@
 # Compute number of observations per year
 # packages: lubridate, tidyverse
 tally_site_obs <- function(site_data) {
+  browser()
```

Comment:
Don't forget to delete this lil' helper fxn since it is only for debugging!

Reply:
Oh glad you're paying attention! :P